src/components/badges/_badges.scss

```diff
@@ -15,7 +15,7 @@ $includeHtml: false !default;
     @include fixText($badgeFontSize);
     @include uppercaseText;
     overflow: visible; // allow badges align as text
-    min-height: auto;
```

Comment:
According to MDN it is: https://developer.mozilla.org/en-US/docs/Web/CSS/min-height

But not supported by IE and Safari.
Seems fair for me to call it incorrect :)